base/strings/basic.jl

```diff
@@ -403,6 +403,25 @@ julia> lowercase("STRINGS AND THINGS")
 lowercase(s::AbstractString) = map(lowercase, s)
 
 """
+    titlecase(s::AbstractString)
+
+Returns `s` with all initial characters converted to titlecase.
```

Comment:
Could you define "initial character"? Maybe also explain what's exactly titlecase.